pkg/reconciler/common/releases.go

```diff
@@ -85,9 +98,42 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	component := "serving"
+	urlList := []string{}
+	switch instance.(type) {
+	case *v1alpha1.KnativeServing:
+		urlList = ServingYamlNames
+	case *v1alpha1.KnativeEventing:
+		component = "eventing"
+		for _, value := range EventingYamlNames {
+			urlList = append(urlList, value)
+		}
```

Comment:
Why not just the same line as above? `urlList = EventingYamlNames`

Reply:
Oops, EventingYamlNames changed.
Thx.